src/table/components/styles.ts

```diff
@@ -18,13 +18,14 @@ export const TableAnnouncer = styled('div')({
 // ---------- TableWrapper ----------
 
 export const StyledTableWrapper = styled(Paper, {
-  shouldForwardProp: (prop: string) => prop !== 'tableTheme' && prop !== 'paginationNeeded',
-})(({ tableTheme, paginationNeeded }) => ({
+  shouldForwardProp: (prop: string) => prop !== 'background' && prop !== 'paginationNeeded',
+})(({ background, paginationNeeded }) => ({
   borderWidth: paginationNeeded ? '0px 1px 0px' : '0px',
   borderStyle: 'solid',
-  borderColor: tableTheme.borderColor,
+  borderColor: background.isDark ? '#F2F2F2' : '#D9D9D9',
```

Comment:
This is temporary. The wrapper wont have a border when we apply the new styling

Reply:
I think you can use this function here: https://github.com/qlik-oss/sn-table/blob/d5bb1cf994a1f9f6f69bc96b15f47fd826e002f3/src/table/utils/styling-utils.ts#L51

Reply:
yeah, but i will remove it soon. There will be no border at all, that is why I did not bother